Makefile

```diff
@@ -50,3 +50,8 @@ format:
 
 coverage:
 	$(ECHO) $(ROOT)/hack/coverage.sh --html "$(COVERAGE)" $(addprefix ./,$(DIRS))
+
+.PHONY: apiserver
+apiserver:
+	go install -v github.com/kubernetes-incubator/service-catalog/cmd/service-catalog
```

Comment:
I'm not sure I understand why we need to do this. won't glide install the dependencies?

Also line 57 feels a bit unexpected.

Do we need Docker image build, etc.?